ape_aws/client.py

```diff
@@ -67,10 +73,76 @@ class CreateKey(CreateKeyModel):
     origin: str = Field(default="AWS_KMS", alias="Origin")
 
 
-class ImportKey(CreateKeyModel):
+class ImportKeyRequest(CreateKeyModel):
     origin: str = Field(default="EXTERNAL", alias="Origin")
 
 
+class ImportKey(ImportKeyRequest):
+    key_id: str = Field(default=None, alias="KeyId")
+    public_key: bytes = Field(default=None, alias="PublicKey")
+    private_key: bytes | None = Field(default=None, alias="PrivateKey")
+    import_token: bytes = Field(default=None, alias="ImportToken")
+
+    @field_validator("private_key")
+    def validate_private_key(cls, value):
+        if not isinstance(value, bytes):
+            return ec.generate_private_key(
+                ec.SECP256K1(),
+                default_backend()
+            )
```

Comment:
Not sure why it is generating a key like this

Reply:
I defined this to work with aws kms at the moment. They used openssl in their documentation. I'm just getting something to work, we can reverse engineer it after

Reply:
Is this the import key?

I don't think we want to generate any keys for the import step

Reply:
I gave the user the ability to allow us to create their private key for them, or they can import a key they already have. Either way, it gives them the ability to store that private key in the `.ape/` directory. If we don't want them to be able to generate a key at all, I can remove this, but I think it's a nice convenience tool. It allows users to playground with an imported key that was generated from our tool, without them having to do any prior work generating that key. Completely up to you

Reply:
Would like this removed. The default behavior is that KMS generates a key for you, which is the highest security since that key is never exposed outside of the service, even in an encrypted format

If you want to import another key you have, you can do that. Importing via mnemonic seems useful too.

Reply:
```python
        .. CAUTION:: For the love of Bob please do not use this mnemonic,
                     it is for testing purposes only.
```
Just found this note in eth_account's docs